internal/store/gorm.go

```diff
@@ -48,6 +49,18 @@ func InitDB(cfg *config.Config, log *logrus.Logger) (*gorm.DB, error) {
 		return nil, err
 	}
 
+	err = newDB.Use(prometheus.New(prometheus.Config{
+		DBName:          cfg.Database.Name,
+		RefreshInterval: 5,
+		StartServer:     true,
+		HTTPServerPort:  15691,
+	}))
+
```

Comment:
should this be optional?

Reply:
Maybe it is on by default which is fine if that is the case

Reply:
Why would someone want to disable it? Would it be for security purposes?

Reply:
I guess if they are not using monitoring and want to reduce the overhead as an example. I can't say its a stick requirement just a note maybe a `TODO: should we allow this to be disabled?`

Reply:
Ok. In that case should I make all three targets (API server, DB, RabbitMQ) optional?

Reply:
For now, lets add a TODO to track